File: lang/ruby/src/vac.fun/algorithms/meta/Sorting.rb
require "vac.fun/algorithms/sorting/BubbleSort"
require "vac.fun/algorithms/sorting/InsertionSort"
require "vac.fun/algorithms/sorting/MergeSort"
require "vac.fun/algorithms/sorting/QuickSort"
require "vac.fun/algorithms/sorting/SelectionSort"

module Sorting

    def self.bubbleSort(arr)
        BubbleSort.sort(arr)
    end

    def self.insertionSort(arr)
        InsertionSort.sort(arr)
    end

    def self.mergeSort(arr)
        MergeSort.sort(arr)
    end

    def self.quickSort(arr)
        QuickSort.sort(arr)
    end

    def self.selectionSort(arr)
        SelectionSort.sort(arr)
    end
end

File: lang/nodejs/src/vac.fun/algorithms/meta/Sorting.js
"use strict"

const BubbleSort    = require("vac.fun/algorithms/sorting/BubbleSort")
const InsertionSort = require("vac.fun/algorithms/sorting/InsertionSort")
const MergeSort     = require("vac.fun/algorithms/sorting/MergeSort")
const QuickSort     = require("vac.fun/algorithms/sorting/QuickSort")
const SelectionSort = require("vac.fun/algorithms/sorting/SelectionSort")

exports.bubbleSort = (arr) => {
    BubbleSort.sort(arr)
}

exports.insertionSort = (arr) => {
    InsertionSort.sort(arr)
}

exports.mergeSort = (arr) => {
    MergeSort.sort(arr)
}

exports.quickSort = (arr) => {
    QuickSort.sort(arr)
}

exports.selectionSort = (arr) => {
    SelectionSort.sort(arr)
}
